packages/components/src/RadarChart/RadarChart.component.js

```diff
@@ -136,10 +136,47 @@ Dot.propTypes = {
 	index: PropTypes.number,
 };
 
+/**
+ * This function provides a clickable axis dot element
+ * @param {Object} props the current props of the Radar
+ */
+// TODO 6.0: remove this export
+export function DotWithClick(props) {
+	const { activeAxis, fill, index, ...rest } = props;
```

Comment:
For me, `onClick` should be part of this spread and listed in the proptypes. It could improve understanding.

Reply:
onClick now included in component param spread.